### ISSUE TITLE
Button to switch between day and night

### ISSUE THREAD

**Issue by ChrissW-R1** [webbukkit/dynmap#3914]
**Feature Description:** As a user I want to have an opportunity to switch between day and night.

**Additional context:**
* Currently the user is not able to see the day map at night and vice-versa.
* A possible solution could be found here: https://www.reddit.com/r/Dynmap/comments/s6pv8w/toggle_button_for_nightday_map/

**Comment by JurgenKuyper** [comment]
you can add seperate maps rendered in day and in night mode, then you can switch manually, the other option is to add an day and night map, so it switches according to the server time

**Comment by ChrissW-R1** [comment]
Yes, this is possible. But I want both: In general the map should be changed by server time. But if the user wants to change to see the day map on night, it should be possible.

It is possible to create three maps: night-and-day, day and night. But with this I had to store every tile double times.

**Comment by JurgenKuyper** [comment]
I am not sure you have to, what happens if you define a map with day and night and then a map with only day with the same prefix?
